### DISCUSSION TITLE
How to deal with JSON ConvertedType?

### DISCUSSION THREAD

**Discussion by gitChristian** [duckdb/duckdb#5346] [Q&A]
I'm trying to import a parquet file into duckdb using the nodejs sdk:
`SELECT * read_parquet('s3://bucket/my.parquet')`

This parquet file has columns with JSON as the logical type. Is this not supported?
I receive this error `Error: Unsupported converted type`

Would you recommend just annotating this as a UTF8 type instead?
I can change the format of the parquet file if need be.

**Comment by Mytherin** [comment]
Could you share a Parquet file that has this property set? We should be able to implement the `JSON` type there. As a work-around you can indeed annotate the type with `UTF8` instead.

**Reply by gitChristian** [reply]
Ok great thank you!
Github doesnt support uploading parquet files, so I put it on gdrive
https://drive.google.com/file/d/1I5xcxQQkL29rTgF7f77ThF0q_1zQRpph/view?usp=sharing

**Reply by gitChristian** [reply]
Please let me know if there's anything else I can do to help.

**Comment by gitChristian** [comment]
How can I best keep this on your guy's radar? Your work is much appreciated!

**Reply by Mytherin** [reply]
It's already fixed in #5401
